src/sentry/search/events/fields.py

```diff
@@ -473,6 +465,23 @@ def resolve_field_list(
                         if function.details.instance.redundant_grouping:
                             aggregate_fields[format_column_as_key(function.aggregate[1])].add(field)
 
+    snuba_filter_condition_aggregates = (
+        set(snuba_filter.condition_aggregates or []) if check_auto_aggregates else set()
+    )
+    for field in set(fields[:]).union(snuba_filter_condition_aggregates):
```

Comment:
Since we're checking here, we could check `aggregations` or `aggregate_fields` instead right?

Reply:
hmm I'm not sure- it seems like `condition_aggregates` is a subset of `aggregations` - so could that have unintended consequences

Reply:
It won't work 😞 
```
["apdex(multiIf(equals(tupleElement(project_threshold_config,1),'lcp'),if(has(measurements.key,'lcp'),arrayElement(measurements.value,indexOf(measurements.key,'lcp')),NULL),duration),tupleElement(project_threshold_config,2))", None, 'apdex']
```
an aggregate looks like this, and we can't tell if this is using the legacy apdex or new apdex unless we check the value of the first element (the function string), so I think we just may have to use fields + conditional aggregates